module/.jest/__tests__/hooks/useHasTimeElapsed.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useHasTimeElapsed, useHasTimeElapsedSinceMount } from '../../../src/hooks/useHasTimeElapsed';
```

Comment:
Could tests live side by side with the hooks somehow?

Reply:
Turns out the answer is yes - will push this out in the next commit. Sadly Cypress still has to go on its own naughty step unless we go to the extent of multiple tsconfigs, one per folder in src to get the right flavour of tests.

Reply:
Found someone else annoyed enough by Cypress being greedy to provide a fix for this issue.